src/javascripts/Services/columnsService.js

```diff
@@ -96,26 +63,22 @@ ngapp.service('columnsService', function(settingsService, nodeHelpers) {
         }
     };
 
-    let defaultColumnsConfig = {
-        customColumns: [],
-        activeColumns: [{
-            label: 'FormID',
-            width: '40%'
-        }, {
-            label: 'EditorID',
-            width: '25%'
-        }, {
-            label: 'Name'
-        }]
-    };
-
     this.loadColumns = function() {
-        let data = fh.loadJsonFile('columns.json') || defaultColumnsConfig;
-        service.columns = [formIDColumn, editorIDColumn, nameColumn];
-        data.customColumns.forEach((column) => service.addColumn(column));
-        data.activeColumns.forEach((column) => service.setColumnData(column));
+        let fileData = fh.loadJsonFile('columns.json');
```

Comment:
I'm wondering if it would make sense to have separate JSON files for each view's columns?  Something worth considering.